src/main/java/com/example/demo/exception/EmailException.java

```diff
@@ -0,0 +1,17 @@
+package com.example.demo.exception;
+
+import javax.mail.internet.AddressException;
+import javax.mail.internet.InternetAddress;
+
+public class EmailException {
```

Comment:
The name is not correspond to the logic this class encapsulates. Please, change to something like `EmailValidator`

Reply:
Ok, i changed.